robottelo/ui/base.py

```diff
@@ -126,12 +132,12 @@ def search(self, element_name):
         strategy, value = element_locator
         strategy2, value2 = common_locators['select_filtered_entity']
         for _ in range(self.result_timeout):
-            element = self.find_element((strategy, value % element_name))
-            if element is not None:
-                return element
-            element2 = self.find_element((strategy2, value2 % element_name))
-            if element2 is not None:
-                return element2
+            result = self.find_element((strategy, value % element))
```

Comment:
Not sure why `element_name` was substituted on `element`, because first line of code in that method suggest to use `element_name` instead. Could you please, explain that?